sla_per_user_system_test.py

```diff
@@ -103,6 +107,13 @@ def validate_scheduler_runtime(self, start_time, end_time, read_users, expected_
 
             shards_time_per_sla = self.prometheus_stats.get_scylla_scheduler_runtime_ms(start_time, end_time, node_ip)
             if not (shards_time_per_sla and scheduler_shares):
+                # Set this message as WARNING because I found that prometheus return empty answer despite the data
```

Comment:
Do we get empty data on every request or is this more of a timing issue? It's really strange to get different results than with a manual query here.

Reply:
@KnifeyMoloko 
yes, it's wierd, agree. It maybe network (connection) issue.
I can add retrying decorator here, but not sure we need it

Reply:
I'm more concerned if this check will add anything if it keeps failing to get the data on every run. Maybe we can create a task to take a look at this once we have a few runs with your PR merged and either change this or remove it. What do you think?

Reply:
@KnifeyMoloko 
I commented this part and added `TODO` to follow if this issue still happens.
Is it OK?

Reply:
Works for me :)